.github/workflows/scheduled-link-check.yml

```diff
@@ -0,0 +1,17 @@
+name: Scheduled Link Check
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "0 0 1 * *"
```

Comment:
We want to change the timing per repo not to run all of the link checkers at the same time and get caught in the rate limit. This is now same as in metal3-docs.